src/pybamm/solvers/base_solver.py

```diff
@@ -65,6 +65,7 @@ def __init__(
         self.name = "Base solver"
         self.ode_solver = False
         self.algebraic_solver = False
+        self._supports_interp = False
```

Comment:
make this a property instead (and we should have made `ode_solver` and `algebraic_solver` properties too so might as well do that while you're there)